beacon_node/network/src/service.rs

```diff
@@ -114,6 +136,26 @@ impl<T: BeaconChainTypes> Service<T> {
     pub fn libp2p_service(&self) -> Arc<Mutex<LibP2PService>> {
         self.libp2p_service.clone()
     }
+
+    /// Attempt to persist the enrs in the DHT to `self.store`.
+    pub fn persist_dht(&self) -> Result<(), store::Error> {
+        let enrs: Vec<Enr> = self
+            .libp2p_service()
+            .lock()
+            .swarm
+            .discovery_mut()
```

Comment:
`enr_entries()` is now built into the behaviour, so there is no need here to get a mutable reference to the actual discovery behaviour.